app/views/foreman/smart_proxies/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<% if @smart_proxy.has_feature?(SmartProxy::PULP_NODE_FEATURE) -%>
+  <div ng-controller="CapsuleContentController">
+    <%= render :file => 'smart_proxies/show' %>
```

Comment:
Nope, it will always render foreman's show but wrapped with ng-controller for pulp nodes. That's needed to get the angular code working.